apps/rush-lib/src/logic/EventHooksManager.ts

```diff
@@ -45,8 +45,10 @@ export class EventHooksManager {
             }
           );
         } catch (error) {
-          console.error(`${os.EOL} Event hook "${script}" failed. Run "rush" with --debug` +
-            ` to see detailed error information.`);
+          console.error(
+            os.EOL + colors.yellow(`Event hook "${script}" failed. Run "rush" with --debug`) +
```

Comment:
Can you make the ` to see detailed error information.` part yellow too?

Reply:
Done sorry, made a mistake when removing prettier auto-correct :/